lib/connextclient/ConnextClient.ts

```diff
@@ -324,7 +329,7 @@ class ConnextClient extends SwapClient {
         tokenAddress = this.tokenAddresses.get(deal.takerCurrency)!;
         const executeTransfer = this.executeHashLockTransfer({
           assetId: tokenAddress,
-          amount: `${amount}`,
+          amount: BigInt(amount).toString(),
```

Comment:
I think it makes sense to do this BigInt conversion where we initialize a value for amount a few lines above, and declare amount as string.

Reply:
Good idea, changed.